machine/v1beta1/types_vsphereprovider.go

```diff
@@ -193,13 +193,34 @@ type VSphereDisk struct {
 	// +required
 	Name string `json:"name"`
 	// sizeGiB is the size of the disk in GiB.
-	// The maximum supported size is 57742 GiB.
+	// The maximum supported size is 16384 GiB.
 	// +kubebuilder:validation:Minimum=1
-	// +kubebuilder:validation:Maximum=57742
+	// +kubebuilder:validation:Maximum=16384
 	// +required
 	SizeGiB int32 `json:"sizeGiB"`
+	// provisioningType specifies the provisioning type to be used by default for VirtualMachine volumes exclusively
+	// owned by this VirtualMachine. This does not apply to PersistentVolumeClaim volumes that are created and managed externally.
+	// If not set, the setting will be provided by the default storage policy.
+	// +optional
+	ProvisioningType ProvisioningType `json:"provisioningType,omitempty"`
```

Comment:
`ProvisioningType` is structured like a field with enum constraints. Is this field supposed to be constrained to specific set of values?